apps/fhir/bluebutton/utils.py

```diff
@@ -187,6 +187,13 @@ def set_default_header(request, header=None):
 
     header['keep-alive'] = settings.REQUEST_EOB_KEEP_ALIVE
 
+    if request.is_secure:
+        header['X-Forwarded-Proto'] = "https"
+    else:
+        header['X-Forwarded-Proto'] = "http"
+
+    header['X-Forwarded-Host'] = settings.HOSTNAME_URL + settings.FHIR_API_PATH
```

Comment:
I think FHIR_API_PATH should be hardcoded as /v1/fhir. The matching URLs are hardcoded in urls.py, and this would be a major breaking change for our partners. There's no need for this to be configurable